test/nodes/test_llm_gemini.py

```
@@ -0,0 +1,413 @@
+        status, emsg, code = ig._extract_status_message_code('', None)
+        assert status is None
+        assert emsg is None
+        assert code is None
+
+    def test_extract_uses_prov_code_fallback(self):
+        """Should use prov_code if no code found in string."""
+        ig = self._make_iglobal()
+        # Use a format where the JSON is a single top-level dict (no nested braces)
+        # so rfind('{') finds the right starting point.
+        raw = '503 UNAVAILABLE. Service is unavailable'
+        status, emsg, code = ig._extract_status_message_code(raw, 503)
```

Ruff (0.15.7)

[warning] Unpacked variable `emsg` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)